internal/linker/linker.go

```diff
@@ -24,6 +24,7 @@ import (
 
 const (
 	MagicValueEnv = "GARBLE_LINKER_MAGIC"
+	TinyEnv       = "GARBLE_LINK_TINY"
```

Comment:
nit: be consistent about `LINK` vs `LINKER` :) I think `LINK` for both is fine, given the program is `link`